src/index.js

```diff
@@ -219,6 +221,9 @@ function addMetadata({ types: t }) {
         state.opts.setupCall;
         state.opts.moduleFunction;
         state.opts.hooksIdentifier;
+        state.opts.getTestMetadataUID =
+          babelPath.scope.generateUidIdentifier(GET_TEST_METADATA);
+        state.opts.contextIdentifier = t.identifier(CONTEXT);
```

Comment:
is there any reason to store this on state, other than to use it in `getTestMetadataDeclaration`?